README.md

```diff
@@ -64,10 +64,11 @@ end
 ## Shorteners
 
 Embiggen ships with a default list of URL shortening service domains (c.f.
-[Acknowledgements](#acknowledgements)) in
-[`shorteners.txt`](https://github.com/altmetric/embiggen/blob/master/shorteners.txt)
-but as it is likely to be outdated and incomplete, you are strongly encouraged
-to supply your own via `Embiggen.configure`.
+[Acknowledgements](#acknowledgements)) in [`shorteners.txt`][SHORTENERS].
+It includes several free domains, but also 1K+ branded shortener domains.
```

Comment:
Rather than adding this here, should we expand on it in the "Acknowledgements" section instead (also we don't need to use all-caps for the [Markdown link label](https://daringfireball.net/projects/markdown/syntax#link), we could just use `shorteners`).

Perhaps mention that it's a combination of the LongURL.org, Bit.do and Hongkiat curated lists (if it was) and a list of Bitly branded domains collected by Altmetric?